README.md

```diff
@@ -259,8 +259,7 @@ RUN add-tool-shed --url 'http://testtoolshed.g2.bx.psu.edu/' --name 'Test Tool S
 RUN install-biojs msa
 
 # Install deepTools
-RUN install-repository \
-    "--url https://toolshed.g2.bx.psu.edu/ -o bgruening --name deeptools"
+RUN install-tools my_tool_list.yml
```

Comment:
I was actually trying to avoid that because it would need to use the install script directly vs. going through the `install-tools` wrapper and Ansible. Let me see how *messy* it gets and will then post a comment for opinions.